### DISCUSSION TITLE
v0.8.0 - Rich Customization

### DISCUSSION THREAD

**Discussion by neon-age** [neon-age/Smart-Inspector#10] [Announcements]
Full Changelog: #9 

### Preferences Window!
- Disable plugin or individual patches
- Hide "Script" fields
- Hide header buttons
- IMGUI Culling - skips GUI for performance!
- Compact Prefab Inspector!
- Compact Scrollbar
- Smooth Scrolling
- Hide Addressable, Convert To Entity
- Hide Asset Labels, Asset Bundle

### Fluent UITK
- API changes, move into AV.UITK namespace
- Will be moved into separate package soon and used as a dependency (!)
- New Controls
- Better Styling
- More Callbacks
- Binding Utils

And some important bug fixes...

<hr /><em>This discussion was created from the release <a href='https://github.com/neon-age/Smart-Inspector/releases/tag/v0.8.0'>v0.8.0 - Rich Customization</a>.</em>

**Comment by Siberdt** [comment]
> Preferences Window!

Hmmm... but how to open the Preferences Window?

**Reply by neon-age** [reply]
Edit > Preferences... > Open Labs > Smart Inspector

**Reply by Siberdt** [reply]
Oh, thanks for the answer, but I have no this option. (v2021.1.14)

UPD: Got it! The version installed from PackageManager was 0.6.0, and my eyes decided it was 0.8.0
Now all works fine. Thank you.

**Reply by neon-age** [reply]
It was a problem with OpenUPM - the v0.8.0 build failed for no reason. Fixed today :)
